### PR TITLE
chore: adjust CI change detection file paths after file restructure

### PR DESCRIPTION
**Pull request by AgustinBettati**
## Description

Adjust change detection paths after merged PR https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1657

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [x] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code
- [x] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.

### CODE REVIEW

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: .github/workflows/acceptance-tests.yml [hunk 1]
```diff
@@ -64,11 +64,11 @@ jobs:
           backup_schedule:
             - 'mongodbatlas/**backup_schedule**.go'
           project:
+            - 'internal/service/project/*.go'
+            - 'internal/service/projectipaccesslist/*.go'
             - 'mongodbatlas/data_source_mongodbatlas_project_invitation*.go'
-            - 'mongodbatlas/fw_data_source_mongodbatlas_project_ip_access_list*.go'
             - 'mongodbatlas/resource_mongodbatlas_access_list_api_key*.go'
             - 'mongodbatlas/resource_mongodbatlas_project_invitation*.go'
```
Comment (lantoli) [inline .github/workflows/acceptance-tests.yml]: you need to remove the "mongodbatlas" part from files

**Review by lantoli — CHANGES_REQUESTED** [review]
you need to remove the "mongodbatlas" part from files

File: .github/workflows/acceptance-tests.yml (hunk 1 above)
Comment (lantoli) [reply .github/workflows/acceptance-tests.yml]: or use like in other places like: 'mongodbatlas/ * * access_list_api_key * *.go' , etc.

(2 asterisks together make bold :-) )

Comment (AgustinBettati) [reply .github/workflows/acceptance-tests.yml]: good point, had forgotten about that change

**Review by lantoli — APPROVED** [review]
LGTM, thanks!

**Pull request merged**